non_interactive_map.R

```diff
@@ -0,0 +1,95 @@
+install.packages("ggmap")
+install.packages("readr")
+install.packages("tidyr")
+install.packages("plotly")
+
+
+library(ggmap)
+library(maptools)
+library(maps)
+
+library (readr)
+library(tidyr)
+library(plyr)
+
+
+results <- read_csv("C:/Users/tonyn/Downloads/survey_results_public.csv")
+View(results)
+#importing country codes Paul uploaded to github
+country_codes<- read_csv("C:/Users/tonyn/Downloads/countries.csv")
+#cleaning up data
+country_codes <- country_codes[c(-1)]

```

Comment:
generally good practice to use different names that suggest meaning or intent.
so country_code_csv might be a better name on line 19. 

But 21, country_codes is probably fair in this regard as it has stripped away the columns that it is not interested in.